.github/workflows/release.yml

```diff
@@ -90,6 +90,21 @@ jobs:
           export GITHUB_TOKEN=${{ secrets.CHE_BOT_GITHUB_TOKEN }}
 
           ./make-release.sh --version ${{ github.event.inputs.version}} --trigger-release
+      - name: Extract content for publishing
```

Comment:
Do we want to also add nightly (on push) publish to GitHub pages? Or GitHub pages = released version, nightly = surge.sh?

Reply:
yes we could do that. maybe in a second  PR? also planning to improve the dogfooding devfile using the same script.

Reply:
:+1: